common/models/acl.js

```diff
@@ -506,12 +506,19 @@ module.exports = function(ACL) {
   };
 
   ACL.resolveRelatedModels = function() {
-    if (!this.roleModel) {
-      var reg = this.registry;
-      this.roleModel = reg.getModelByType(loopback.Role);
-      this.roleMappingModel = reg.getModelByType(loopback.RoleMapping);
-      this.userModel = reg.getModelByType(loopback.User);
-      this.applicationModel = reg.getModelByType(loopback.Application);
+    var self = this;
+    if (!self.roleModel) {
+      var reg = self.registry;
+      var builtInModels = {
+        roleModel: reg.getModel('Role'),
+        roleMappingModel: reg.getModel('RoleMapping'),
+        userModel: reg.getModel('User'),
+        applicationModel: reg.getModel('Application'),
+      };
+
+      for (var m in builtInModels) {
+        self[m] = reg.getModelByType(builtInModels[m]);
```

Comment:
`getModelByType` works for string names too, no need to call `reg.getModel` first.

See https://github.com/strongloop/loopback/blob/21ce174939843aef15dab28dad353a931e086c4f/lib/registry.js#L324-L326